controllers/database_controller.go

```diff
@@ -577,57 +576,18 @@ func (r *DatabaseReconciler) createProxy(ctx context.Context, dbcr *kciv1alpha1.
 	return nil
 }
 
-func (r *DatabaseReconciler) createTemplatedSecrets(ctx context.Context, dbcr *kciv1alpha1.Database, ownership []metav1.OwnerReference) error {
+func (r *DatabaseReconciler) createTemplatedSecrets(ctx context.Context, dbcr *kciv1beta1.Database, ownership []metav1.OwnerReference) error {
 	// First of all the password should be taken from secret because it's not stored anywhere else
 	databaseSecret, err := r.getDatabaseSecret(ctx, dbcr)
 	if err != nil {
 		return err
 	}
-	// Then parse the secret to get the password
```

Comment:
just a note: important to document in release note

Reply:
I'm sorry, is it about removing of the connection string generator? There is only one line highlighted, so I'm not really sure